### PR TITLE
[Project] Add ACCV workshop 1st Solution

### PR DESCRIPTION
**Pull request by Ezra-Yu**
## Motivation

Add ACCV workshop 1st Solution

### CODE REVIEW

File: projects/fgia_accv2022_1st/README.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
+# Solution of FGIA ACCV 2022(1st Place)
+
+This is the 1st Place Solution for Webly-supervised Fine-grained Recognition, refer to the ACCV workshop competition in https://www.cvmart.net/race/10412/base.
```
Comment (YuanLiuuuuuu) [inline projects/fgia_accv2022_1st/README.md:3]: Better indicate this is the fine-tuning solution. For pre-training, you can put a link pointing to MMSelfSup.

File: projects/fgia_accv2022_1st/README.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+
+![LB-B](https://user-images.githubusercontent.com/18586273/205498171-5a3a3055-370a-4a8b-9779-b686254ebc94.png)
+
+</br>
+
+</details>
+
+## Reproduce / 复现
+
+For detailed self-supervised pretrain code, please refer to [MMselfSup](https://github.com/open-mmlab/mmselfsup/tree/dev-1.x/projects/fgia_accv2022_1st).
```
Comment (YuanLiuuuuuu) [inline projects/fgia_accv2022_1st/README.md:27]:
```suggestion
For detailed self-supervised pretrain code, please refer to [MMSelfSup](https://github.com/open-mmlab/mmselfsup/tree/dev-1.x/projects/fgia_accv2022_1st).
```

File: projects/fgia_accv2022_1st/README.md [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+For detailed finetuning and inference code, refer to [this repo](https://github.com/Ezra-Yu/ACCV2022_FGIA_1st).
+
+## Description
+
+### Overview Flow
```
Comment (YuanLiuuuuuu) [inline projects/fgia_accv2022_1st/README.md:32]:
```suggestion
### Overview of Our Solution
```

File: projects/fgia_accv2022_1st/README.md [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+
+![image](https://user-images.githubusercontent.com/18586273/205498371-31dbc1f4-5814-44bc-904a-f0d32515c7dd.png)
+
+### Model Select
```
Comment (YuanLiuuuuuu) [inline projects/fgia_accv2022_1st/README.md:36]:
```suggestion
### Our Model
```

File: projects/fgia_accv2022_1st/README.md [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+
+- ViT(MAE-pt)   # Pretrained from [**MMSelfSup**](https://github.com/open-mmlab/mmselfsup/tree/dev-1.x/projects/fgia_accv2022_1st).
```
Comment (YuanLiuuuuuu) [inline projects/fgia_accv2022_1st/README.md:38]:
```suggestion
- ViT(MAE-pre-train)   # Pretrained from [**MMSelfSup**](https://github.com/open-mmlab/mmselfsup/tree/dev-1.x/projects/fgia_accv2022_1st).
```

File: projects/fgia_accv2022_1st/README.md [hunk 6]
```diff
@@ -0,0 +1,76 @@
...
+- Swin(21kpt)   # From [MMCls-swin_transformer_v2](https://github.com/open-mmlab/mmclassification/tree/dev-1.x/configs/swin_transformer_v2).
```
Comment (YuanLiuuuuuu) [inline projects/fgia_accv2022_1st/README.md:39]:
```suggestion
- Swin-v2(SimMIM-pre-train)   # From [MMCls-swin_transformer_v2](https://github.com/open-mmlab/mmclassification/tree/dev-1.x/configs/swin_transformer_v2).
```

File: projects/fgia_accv2022_1st/README.md (hunk 1 above)
Comment (Ezra-Yu) [reply projects/fgia_accv2022_1st/README.md]: done

File: projects/fgia_accv2022_1st/README.md [hunk 7]
```diff
@@ -0,0 +1,76 @@
...
+
+**Main Arch**
```
Comment (YuanLiuuuuuu) [inline projects/fgia_accv2022_1st/README.md:41]:
```suggestion
**The architectures we use **
```

File: projects/fgia_accv2022_1st/README.md [hunk 8]
```diff
@@ -0,0 +1,76 @@
...
+- self-emsemble: [Uniform-model-soup](https://arxiv.org/abs/2203.05482) | [code](https://github.com/Ezra-Yu/ACCV_workshop/blob/master/tools/model_soup.py)
+- [pseudo](https://lilianweng.github.io/posts/2021-12-05-semi-supervised/)  | [Code](https://github.com/Ezra-Yu/ACCV_workshop/blob/master/tools/creat_pseudo.py)
+- bagging-emsemble [Code](https://github.com/Ezra-Yu/ACCV_workshop/blob/master/tools/emsemble.py),
+- post-process: [re-distribute-label](https://github.com/Ezra-Yu/ACCV_workshop/blob/master/tools/re-distribute-label.py);
+
+![Overview](https://user-images.githubusercontent.com/18586273/205498258-e5720d83-7006-4aea-86b5-aab1a8998c6c.png)
+
+![image](https://user-images.githubusercontent.com/18586273/205498027-def99b0d-a99a-470b-b292-8d5fc83111fc.png)
+
+#### Used but no improvement
```
Comment (YuanLiuuuuuu) [inline projects/fgia_accv2022_1st/README.md:67]:
```suggestion
#### Used but no improvements
```

File: projects/fgia_accv2022_1st/README.md [hunk 9]
```diff
@@ -0,0 +1,76 @@
...
+
+1. Retrieval Paradigm
```
Comment (YuanLiuuuuuu) [inline projects/fgia_accv2022_1st/README.md:69]: Cluster-based data filtering?

File: projects/fgia_accv2022_1st/README.md [hunk 10]
```diff
@@ -0,0 +1,76 @@
...
+2. EfficientNetv2
+
+#### Not used but worth to do
+
+1. DiVE for Long tail Problem;
```
Comment (YuanLiuuuuuu) [inline projects/fgia_accv2022_1st/README.md:74]: What does ``Dive for`` mean?

File: projects/fgia_accv2022_1st/README.md (hunk 9 above)
Comment (Ezra-Yu) [reply projects/fgia_accv2022_1st/README.md:69]: no, After extracting the features, compare them in the training set and find the most similar samples in the train

File: projects/fgia_accv2022_1st/README.md [hunk 11]
```diff
@@ -0,0 +1,76 @@
...
+2. Use SimMIM to train a swinv2 SelfSup pretrined model;
```
Comment (YuanLiuuuuuu) [inline projects/fgia_accv2022_1st/README.md:75]:
```suggestion
2. Use SimMIM to pre-train Swin-v2 on the competition dataset
```

File: projects/fgia_accv2022_1st/README.md [hunk 12]
```diff
@@ -0,0 +1,76 @@
...
+3. improve re-distribute-label tool
```
Comment (YuanLiuuuuuu) [inline projects/fgia_accv2022_1st/README.md:76]:
```suggestion
3. refine the re-distribute-label tool
```

**Review by YuanLiuuuuuu — APPROVED** [review]
LGTM

**Pull request merged**